sklearn/ensemble/iforest.py

```diff
@@ -227,25 +257,9 @@ def predict(self, X):
 
         scores = 2 ** (-depths.mean(axis=1) / _average_path_length(self.max_samples_))
 
-        return scores
-
-    def decision_function(self, X):
-        """Average of the decision functions of the base classifiers.
-
-        Parameters
-        ----------
-        X : {array-like, sparse matrix}, shape (n_samples, n_features)
-            The training input samples. Sparse matrices are accepted only if
-            they are supported by the base estimator.
-
-        Returns
-        -------
-        score : array, shape (n_samples,)
-            The decision function of the input samples.
-
-        """
-        # minus as bigger is better (here less abnormal):
-        return - self.predict(X)
+        # minus as bigger is better (here less abnormal)
+        # and add 0.5 to give a sense to scores = 0:
+        return 0.5 - scores
```

Comment:
this value plays a special role and comes from the original paper. Should I add a comment?